packages/babel-plugin-transform-react-jsx/test/fixtures/react/proto-in-jsx-attribute/input.js

```diff
@@ -0,0 +1 @@
+<p __proto__={null} class="bar">text</p>;
```

Comment:
Both TS and Babel interpret `__proto__` in JSXAttributeKey as the special `__proto__` accessor, though JSX spec does not specify such behaviour.